### PR TITLE
Add arda example

### PR DESCRIPTION
**Pull request by mizchi**
Implement flux-comparison with [mizchi/arda](https://github.com/mizchi/arda)

https://github.com/voronianski/flux-comparison/issues/32

### CODE REVIEW

File: arda/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+          "optional": [
+            "runtime"
+          ]
+        }
+      ],
+      "envify"
+    ]
+  },
+  "dependencies": {
+    "arda": "^0.13.1"
```
Comment (voronianski) [inline arda/package.json]: @mizchi please add `arda` to global dependencies instead - https://github.com/voronianski/flux-comparison/blob/master/package.json#L14

File: arda/js/app.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+global.React = React;
+import Arda from 'arda';
+import AppContext from './contexts/app'
+
+import shop from '../../common/api/shop';
+
+window.addEventListener("DOMContentLoaded", () => {
```
Comment (voronianski) [inline arda/js/app.js]: @mizchi double -> single quotes, see [CONTRIBUTING](https://github.com/voronianski/flux-comparison/blob/master/CONTRIBUTING.md).

File: arda/js/contexts/app.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import Arda from 'arda'
+import AppComponent from '../components/App.jsx'
+
+export default class ArdaContext extends Arda.Context {
+    get subscribers() { return [require('./subscriber')]; }
+    get component() { return require('../components/App.jsx'); }
```
Comment (voronianski) [inline arda/js/contexts/app.js]: @mizchi why `require` here? Isn't better and cleaner to make `import`s in the top?

File: arda/index.html [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>Arda Sample</title>
+
+    <link rel="shortcut icon" type="image/png" href="../common/assets/react.png">
+    <link rel="stylesheet" href="../common/css/uikit.almost-flat.min.css">
+    <link rel="stylesheet" href="../common/css/main.css">
+    <script src="build/bundle.js"></script>
```
Comment (voronianski) [inline arda/index.html:10]: @mizchi please add `build/*.js` of arda to [.gitignore](https://github.com/voronianski/flux-comparison/blob/master/.gitignore#L14).

**Comment by mizchi** [comment]
Thx, I will fix soon so

**Comment by voronianski** [comment]
@mizchi I'm closing this for now.. if you want, please rebase with latest master and make PR correspond contribution guidelines, thanks.

**Pull request closed without merging**